.github/workflows/web.yml

```diff
@@ -586,53 +564,28 @@ jobs:
       registry_password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
       composer_token: ${{ secrets.CENTREON_TECHNIQUE_PAT }}
 
-  create-xray-test-plan-and-test-execution:
-    needs: [get-version, dockerize]
-    if: ${{ !cancelled() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') && ( needs.get-version.outputs.stability == 'testing' || github.event_name == 'schedule' ) }}
-    strategy:
-      fail-fast: false
-      matrix:
-        os: ${{ fromJson(needs.get-version.outputs.dockerize_matrix) }}
-
-    uses: ./.github/workflows/create-xray-test-plan-and-test-execution.yml
-    with:
-      major_version: ${{ needs.get-version.outputs.major_version }}
-      minor_version: ${{ needs.get-version.outputs.minor_version }}
-      os: ${{ matrix.os }}
-    secrets:
-      xray_client_id: ${{ secrets.XRAY_CLIENT_ID }}
-      xray_client_secret: ${{ secrets.XRAY_CLIENT_SECRET }}
-      xray_jira_user_email: ${{ secrets.XRAY_JIRA_USER_EMAIL }}
-      xray_jira_token: ${{ secrets.XRAY_JIRA_TOKEN }}
-
-  e2e-test:
-    needs: [get-version, dockerize, create-xray-test-plan-and-test-execution]
-    if: ${{ !cancelled() && !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') && needs.get-version.outputs.stability != 'stable' }}
+  create-xray-test-plan-and-test-execution-e2e-test-newman-test:
```

Comment:
this makes the web workflow less readable
why not adding `create-xray-test-plan-and-test-execution` to the needs of `newman-test`

Reply:
@kduret the problem is not for newman-test, when NIGHTLY is running, we got 3 TEs, TE-1 for alma8, TE-2 for alma9, TE-3 for bullseye, when the workflow e2e-test is executed, it will also be executed 3 times, but with the same input TE-1, instead of having different input each time is running